packages/api/src/api/helix/video/HelixVideo.ts

```diff
@@ -134,6 +134,18 @@ export class HelixVideo extends DataObject<HelixVideoData> {
 		return this[rawDataSymbol].thumbnail_url;
 	}
 
+	/**
+	 * Builds the thumbnail URL of the video using the given dimensions.
+	 *
+	 * @param width The width of the thumbnail.
+	 * @param height The height of the thumbnail.
+	 */
+	getThumbnailUrl(width: number, height: number): string {
+		return this[rawDataSymbol].thumbnail_url
+			.replace('{width}', width.toString())
```

Comment:
This doesn't work - apparently Twitch uses `%{placeholder}` rather than `{placeholder}` in only this specific case.

Reply:
Oh well spotted! Fixed that.